Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -47,18 +48,28 @@ namespace gdjs {
       if (!!this._input)
         throw new Error('Tried to recreate an input while it already exists.');
 
+      this._form = document.createElement('form');
       const isTextArea = this._object.getInputType() === 'text area';
       this._input = document.createElement(isTextArea ? 'textarea' : 'input');
-      this._input.style.border = '1px solid black';
+      this._form.style.border = '0px';
       this._input.autocomplete = 'off';
-      this._input.style.borderRadius = '0px';
+      this._form.style.borderRadius = '0px';
+      this._form.style.backgroundColor = 'transparent';
       this._input.style.backgroundColor = 'white';
-      this._input.style.position = 'absolute';
-      this._input.style.resize = 'none';
-      this._input.style.outline = 'none';
-      this._input.style.pointerEvents = 'auto'; // Element can be clicked/touched.
-      this._input.style.display = 'none'; // Hide while object is being set up.
-      this._input.style.boxSizing = 'border-box'; // Important for iOS, because border is added to width/height.
+      this._input.style.border = '1px solid black';
+      this._form.style.position = 'absolute';
+      this._form.style.resize = 'none';
+      this._form.style.outline = 'none';
```

Comment:
Could you split the lines where you change `this._form` and those where you change `this._input`, it's a bit hard to read at the moment